### PR TITLE
Renaming ContrastBrightness

### PR DESCRIPTION
**Pull request by K4rishma**
The contrast brightness function is renamed to align functions in simsem and phenom with xt_client. 

Please check the two questions added as TODOs in the code.

Testing Method - Respective unit tests were run for all three of them. 
Output               - Tests ran properly for phenom and simsm(passed, ignored)
                          - one test in xt_client failed - TestMBScanner 
                          `    raise TypeError("XTtoolkit must be running to instantiate the multi-beam scanner child.")
TypeError: XTtoolkit must be running to instantiate the multi-beam scanner child.`

### CODE REVIEW

File: install/linux/etc/odemis.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # Use these values during development
-#DEVPATH="$HOME/development"
-#PYTHONPATH="$DEVPATH/odemis/src/:$DEVPATH/Pyro4/src/:$PYTHONPATH"
+DEVPATH="$HOME/development"
+PYTHONPATH="$DEVPATH/odemis/src/:$DEVPATH/Pyro4/src/:$PYTHONPATH"
```
Comment (K4rishma) [inline install/linux/etc/odemis.conf:4]: Do I have to comment this as in the original branch?

File: src/odemis/gui/cont/tabs.py [hunk 2]
```diff
@@ -1068,7 +1068,7 @@ def _onAutofocus(self, active):
                 if self.curr_s.focuser.role == "ebeam-focus" and self.main_data.role == "delphi":
                     self.orig_hfw = emt.horizontalFoV.value
                     emt.horizontalFoV.value = min(self.orig_hfw, AUTOFOCUS_HFW)
-                    f = det.applyAutoContrast()
+                    f = det.applyAutoContrast() #TODO renaming required here?
```
Comment (tepals) [inline src/odemis/gui/cont/tabs.py:1071]: yes

File: src/odemis/gui/cont/settings.py [hunk 3]
```diff
@@ -308,7 +308,7 @@ def adjust_done(_):
         def auto_adjust(_):
             """ Call the auto contrast method on the detector if it's not already running """
             if not btn_autoadjust.up:
-                f = detector.applyAutoContrast()
+                f = detector.applyAutoContrast() # TODO renaming required here?
```
Comment (tepals) [inline src/odemis/gui/cont/settings.py:311]: yes

File: src/odemis/gui/cont/settings.py [hunk 4]
```diff
@@ -308,7 +308,7 @@ def adjust_done(_):
...
                 btn_autoadjust.SetLabel("Adjusting...")
                 btn_autoadjust.Disable()
                 brightness_entry.value_ctrl.Disable()
```
Comment (K4rishma) [inline src/odemis/gui/cont/settings.py:314]: Wasn't sure about renaming this line, so added a TODO.

File: install/linux/etc/odemis.conf (hunk 1 above)
Comment (tepals) [inline install/linux/etc/odemis.conf:4]: you can revert these changes

File: src/odemis/gui/cont/tabs.py (hunk 2 above)
Comment (K4rishma) [inline src/odemis/gui/cont/tabs.py:1071]: Same goes for this, ""Wasn't sure about renaming this line, so added a TODO.""

File: src/odemis/gui/cont/settings.py (hunk 3 above)
Comment (K4rishma) [reply src/odemis/gui/cont/settings.py:311]: Do I have to make changes here? Theses files i.e. delphi.py and dist-packages are read-only. Due to which I do not have editing rights.
image

**Comment by K4rishma** [comment]
@tepals new changes have been done along with tests running.

File: src/odemis/driver/test/phenom_test.py [hunk 5]
```diff
@@ -31,7 +31,6 @@
 import time
 import unittest
 from unittest.case import skip
-
```
Comment (tepals) [inline src/odemis/driver/test/phenom_test.py:34]: no need to remove this empty line, could you add it back?

**Review by tepals — COMMENTED** [review]
one small comment left, the rest looks good!

File: install/linux/etc/odemis.conf (hunk 1 above)
Comment (K4rishma) [reply install/linux/etc/odemis.conf]: Added it back.

File: src/odemis/driver/test/phenom_test.py (hunk 5 above)
Comment (K4rishma) [reply src/odemis/driver/test/phenom_test.py]: Added it back.

**Review by tepals — APPROVED** [review]
